.gitignore

```diff
@@ -18,3 +18,4 @@ venv/
 .DS_Store?
 *.DS_Store
 Thumbs.db
+.idea
```

Comment:
This is a developer-specific thing, so I think it might be better to put it locally in `.git/info/exclude`.